app.py

```diff
@@ -7,6 +7,11 @@
 db = SQLAlchemy()
 r = FlaskRedis()
 auth = Authenticator()
+from model.session import Session
```

Comment:
Model imports are done [here](https://github.com/aueb-cslabs/director/blob/2b8132e12d957bb38660b501de7c734d4281fd8b/app.py#L29), because we need the app to already be initialized when importing them. To mitigate that and to simplify the code, move the imports to the `models/__init.py__` file!